index.js

```diff
@@ -39,6 +39,10 @@ function filterResponseForImportantEvents(allEventsFromFetch) {
         const event = allEventsFromFetch[i];
         if (githubImportantEvents.indexOf(event.type) !== -1) {
             arrayOfImportantEvents.push(event);
+        } else if (event.payload) {
+            if (githubImportantEvents.indexOf(event.type + '.' + event.payload.action) !== -1) {
```

Comment:
This line is fine as it is. My own preference would be to extract a local:
`const typeWithAction = event.type + '.' + event.payload.action`
Which would allow the if to be simpler. Definitely not a required change. Just wanted to pass along a thought for future work.

Reply:
Yeah that would make it simpler to understand. Thanks for suggestion.

Reply:
I haven't run it locally, but visually it looks good. Thanks!